streams/src/main/java/org/apache/kafka/streams/kstream/KGroupedStream.java

```diff
@@ -298,7 +304,9 @@
      * @param storeSupplier user defined state store supplier. Cannot be {@code null}.
      * @return a windowed {@link KTable} that contains "update" records with unmodified keys and {@link Long} values
      * that represent the latest (rolling) count (i.e., number of records) for each key within a window
+     * @deprecated use {@link #windowedBy(Windows)}
```

Comment:
Do we want to add a couple extra words  ` which returns a WindowedKStream enabling count, reduce and aggregate operations` or something along those lines?  

The same goes for the other deprecated aggregation actions.